src/js/angular/repositories/controllers.js

```diff
@@ -362,6 +362,11 @@ function LocationsAndRepositoriesCtrl($scope, $modal, toastr, $repositories, Mod
 
     getLocations();
     const timer = $interval(function () {
+        if (ShepherdService.isActive() && !$rootScope.guidePaused) {
```

Comment:
Add isActive function to GuideService and use it here instead to use it directly from ShepherdService. Idea behaind ShepherdService is to wrapp shepard libraries. 
If we decide to change tour libraries we will implement a new wrapper service and change it only in GuideService.

Reply:
Fixed.